src/net.cpp

```diff
@@ -174,8 +174,7 @@ CAddress GetLocalAddress(const CNetAddr *paddrPeer, ServiceFlags nLocalServices)
 static int GetnScore(const CService& addr)
 {
     LOCK(cs_mapLocalHost);
-    if (mapLocalHost.count(addr) == LOCAL_NONE)
```

Comment:
Huh, this is really strange. I guess it's correct because `LOCAL_NONE` effectively is `0` but it's certainly not right.